Packages/vcs/Lib/VTKPlots.py

```diff
@@ -357,7 +358,7 @@ def createRenderer(self, *args, **kargs):
         return ren
 
     def update(self, *args, **kargs):
-        self._lastSize = -1
+        self._lastSize = None
```

Comment:
@danlipsa have you tested on mac? Mac is VERY different about this, also we need to test within GUI. @chaosphere2112 can you please test on your mac and in your new GUI?